managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -128,6 +127,22 @@ public Result findByName_Corrected(UUID customerUUID, String universeName) {
     return Results.status(OK, Json.toJson(universe.universeUUID));
   }
 
+  /**
+   * API that verify Universe has unique name.
+   *
+   * @return 200 if universe is not present else raise error
+   */
+  public Result verifyUniqueName(UUID customerUUID, String universeName) {
+    // Verify the customer with this universe is present.
+    Customer.getOrBadRequest(customerUUID);
+    LOG.info("Finding Universe with name {}.", universeName);
+    if (Universe.checkIfUniverseExists(universeName)) {
+      throw new YWServiceException(BAD_REQUEST, "Universe already exists");
+    } else {
+      return withMessage("Universe does not Exist");
```

Comment:
Just from code readability POV. Message should be positive sounding since you are returning success. 
"Universe name is unique"

Reply:
good point.